packages/esm-active-visits-app/src/visits-summary/visit-detail.test.tsx

```diff
@@ -36,7 +38,7 @@ describe('VisitDetailComponent', () => {
     render(<VisitDetailComponent visitUuid={visitUuid} patientUuid={patientUuid} />);
 
     expect(screen.getByText(/Some Visit Type/)).toBeInTheDocument();
-    expect(screen.getByText(/29-Jul-2023, 12:34 PM/)).toBeInTheDocument();
+    expect(screen.getByText(formatDate(visitDate))).toBeInTheDocument();
 
```

Comment:
This test started failing after I made the changes described.  But this should not have been passing before if I'm understanding right, because a startDatetime with a "Z" at the end indicates UTC, but my computer running these tests is in EDT, so this shows up as 08:34 AM, not 12:34 PM.  I've just changed the test to use a locally generated date, so whatever is passed is the same as what is tested.  But it _seems_ that this is potentially a fixed bug.